src/debugging/coreclr/dockerManager.ts

```diff
@@ -307,19 +329,26 @@ export class DefaultDockerManager implements DockerManager {
     }
 
     private async getContainerWebEndpoint(containerNameOrId: string): Promise<string | undefined> {
-        const webPorts = await this.dockerClient.inspectObject(containerNameOrId, { format: '{{(index (index .NetworkSettings.Ports \\\"80/tcp\\\") 0).HostPort}}' });
-
-        if (webPorts) {
-            const webPort = webPorts.split('\n')[0];
-
-            // tslint:disable-next-line:no-http-string
-            return `http://localhost:${webPort}`;
+        let portMappingsString = await this.dockerClient.inspectObject(containerNameOrId, { format: '{{json .NetworkSettings.Ports}}' });
+        let portMappings = <IPortMappings>JSON.parse(portMappingsString);
+
+        if (portMappings) {
+            let httpsPort = portMappings["443/tcp"] && portMappings["443/tcp"][0] && portMappings["443/tcp"][0].HostPort || null;
+            let httpPort = portMappings["80/tcp"] && portMappings["80/tcp"][0] && portMappings["80/tcp"][0].HostPort || null;
+
+            if (httpsPort) {
+                return `https://localhost:${httpsPort}`;
```

Comment:
>return [](start = 16, length = 6)

It seems like this ought to be in the launch.json file.

Reply:
Yes, it probably should. I'm going to file an issue and address that in a separate PR.